UPGRADE-3.x.md

```diff
@@ -4,6 +4,21 @@ UPGRADE 3.x
 UPGRADE FROM 3.x to 3.x
 =======================
 
+### Remove deprecated calls
+
+- `Sonata\Doctrine\Model\PageableManagerInterface` is no longer used in profit of `Sonata\DatagridBundle\Pager\PageableInterface`
+
+### Dependencies
+
+- drop support for `sonata-project/datagrid-bundle` < ^3.0.
+
+    If you are extending these method you MUST add argument and return type declarations:
+
+    - `Sonata\MediaBundle\Entity\GalleryManager::getPager()`
+    - `Sonata\MediaBundle\Entity\MediaManager::getPager()`
+    - `Sonata\MediaBundle\Model\MediaManager::getPager()`
+    - `Sonata\MediaBundle\Model\MediaManager::getPager()`
```

Comment:
Why is this method mentioned twice?

Reply:
One if for entity, one is for document.

Reply:
Should be ``Sonata\MediaBundle\Document\MediaManager::getPager()`` then ?

There is no `Sonata\MediaBundle\Model\MediaManager`

Reply:
Oh, correct. I check it in phone and miss mediamanager is twice

Reply:
Done.